ocaml/xapi/dbsync.ml

```diff
@@ -48,6 +48,14 @@ let create_host_metrics ~__context =
     )
     (Db.Host.get_all ~__context)
 
+let update_pool_recommendations ~__context =
+  match Db.Pool.get_all ~__context with
+  | [pool] ->
+      Db.Pool.set_recommendations ~__context ~self:pool
+        ~value:(Xapi_globs.recommendations `Pool)
+  | _ ->
+      failwith "No pool created, or multiple pool objects detected"
```

Comment:
How is this exception being handled? I would instead add a warning logline

Reply:
Could follow the existing style "let pool = Helpers.get_pool ~__context in"?